### PR TITLE
jog out carriage on trap scoring

### PR DESCRIPTION
**Pull request by Lewis-Seiden**
prevents pinching at max extension

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -209,13 +209,15 @@ public void robotInit() {
...
                     .setExtensionCmd(() -> 0.0)
                     .until(() -> elevator.getExtensionMeters() < 0.05),
+                Commands.waitUntil(() -> controller.y().getAsBoolean()),
                 Commands.parallel(
+                    carriage.runVoltageCmd(-CarriageSubsystem.INDEXING_VOLTAGE).withTimeout(0.25),
                     elevator.setExtensionCmd(() -> ElevatorSubsystem.TRAP_EXTENSION_METERS),
                     Commands.waitUntil(
                             () ->
                                 elevator.getExtensionMeters()
                                     > 0.95 * ElevatorSubsystem.TRAP_EXTENSION_METERS)
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/Robot.java:219]: are we going to abstract any of these other ones out?

**Review by Awesomeplayer165 — APPROVED** [review]
overall looks good

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (Awesomeplayer165) [reply src/main/java/frc/robot/Robot.java:219]: meant to highlight `0.95`

Comment (Lewis-Seiden) [reply src/main/java/frc/robot/Robot.java:219]: i wasnt planning on it since most of the others arent reused in so many places

**Pull request merged**